Find a functionally equivalent code:android.view.accessibility.AccessibilityEvent.getRecord:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the record at a given index.
 *
 * @param index The index.
 * @return The record at the specified index.
 */

@@ -1,3 +1,6 @@
 {
+    if (mRecords == null) {
+        throw new IndexOutOfBoundsException("Invalid index " + index + ", size is 0");
+    }
     return mRecords.get(index);
 }

